content/en/docs/distributions/gke/pipelines/preemptible.md

```diff
@@ -49,41 +46,61 @@ The following sections contain more detail about the above steps.
 
 ### 1. Create a node pool with preemptible VMs
 
-Use the `gcloud` command to
-[create a node pool](https://cloud.google.com/sdk/gcloud/reference/container/node-pools/create).
-The following example includes placeholders to illustrate the important
-configurations:
+Create a `preemptible-nodepool.yaml` as below and fulfill all placerholder content:
```

Comment:
Per the styleguide, placeholders must be defined in lower case with angle brackets. Please update the sample code to define the placeholders that you want users top replace.

(https://github.com/kubeflow/pipelines/blob/master/samples/core/preemptible_tpu_gpu/preemptible_tpu_gpu.py)

Reply:
fulfill -> replace

Reply:
It feels easier putting this preemptible-nodepool.yaml inside gcp-blueprints repo as an optional sub-package and refer people to include it in their kustomization.yaml, in that way, we don't need to introduce so many env vars, they'll be just set automatically.

Reply:
I agree, having a file in gcp-blueprint helps a lot with explanation and applying variables with kpt. Since this goes beyond the current PR and repo, let's have a different PR to address this.